src/view/ThemeManager.js

```diff
@@ -238,9 +238,12 @@ define(function (require, exports, module) {
                 })
                 .then(function (cssContent) {
                     $("body").toggleClass("dark", theme.dark);
+                    $("body").toggleClass("light");
                     styleNode.text(cssContent);
-
                     pending.resolve(theme);
+                    if($("body").hasClass("dark") === false && $("body").hasClass("light" ) === false){
```

Comment:
If you want to test that `body` does not have "more than one class" it's actually easier to do that with a single selector:
```
if($("body.dark body.light").length() === 0) {
  // there is no 'dark', or 'light', class attached to the body element
}
```